web/grunt/karma-config.js

```diff
@@ -0,0 +1,37 @@
+module.exports = function (grunt) {
+    'use strict';
+
+    const karmaTasksConfig = {
+        browsers: ['Firefox'],
+        frameworks: ['jasmine', 'browserify'],
+        preprocessors: {
+            '<%= config.tests %>': ['browserify'],
+            '<%= config.compiledScriptsDir %>/**/*.js': ['browserify']
+        },
+        browserify: {
+            debug: true,
+            paths: '<%= config.scriptsDir %>'
+        }
+    };
+
+    const configure = function () {
+        grunt.config.merge({
+            karma: {
+                options: {
+                    files: ['<%= config.compiledScriptsDir %>/**/*.js', '<%= config.tests %>']
+                },
+                dev: karmaTasksConfig,
+                ci: Object.assign(
+                    {
+                        options: {
+                            singleRun: true
+                        }
+                    },
+                    karmaTasksConfig
+                )
+            }
+        });
+    };
+
+    configure();
```

Comment:
haha. Normally I would agree to try to put everything in functions, but since the `configure()` literally only wraps grunt.config.merge()` its almost unnecessary. But I could see how it would be useful if `configure()` was going to do more things.